.bash_profile

```diff
@@ -78,10 +90,12 @@ fi
 #   source ~/.asdf/plugins/java/bin/asdf-java-wrapper
 # fi
 
+ssh-add -kq
```

Comment:
Do you have the `keychain` formula installed? If so, https://github.com/expectedbehavior/common-files/blob/master/.bash_profile#L15-L38 should be handling your ssh key loading. I could see needing to add another keychain path for m1 homebrew, but that wouldn't explain why it didn't work on your old laptop, which made me wonder if you had `keychain` at all.